src/store/ordinoSlice.ts

```diff
@@ -86,51 +102,24 @@ export interface IOrdinoViewPlugin<S extends IBaseState> extends IViewPluginDesc
   state: S;
 }
 
-// const test = ({
-//   headerOverride = Header,
-// }: {
-//   headerOverride?: (props: {view: IView[]}) => JSX.Component;
-//   overrides: {
-//     header: JSX.Component;
-//     itemGroup: JSX.Component;
-//     header: JSX.Component;
-//   }
-// }) => <>
-//   <headerOverride views={views}></headerOverride>
-//   Map( <ListItemGroup>
-//     <ListItem>
-//     ...)
-
-// </>;
-
-// export interface IOrdinoScatterplotViewPlugin<{
-//   color: string;
-// }> {
-//   state: {
-
-//   }
-// }
-
-const containsView = (workbench: IWorkbench, viewId: string) => workbench.views.some(({ uniqueId }) => uniqueId === viewId);
-
 const initialState: IOrdinoAppState = {
   workbenches: [],
-  focusViewIndex: 0,
-  sidebarOpen: false,
+  focusWorkbenchIndex: 0,
   colorMap: {},
 };
 
-// TODO: Change rest of methods to use viewId instead of entity id
 const ordinoSlice = createSlice({
   name: 'ordino',
   initialState,
   reducers: {
+    // TODO in general: does it make sense to group the reducer functions (e.g., by workbench, views, ...)? or even create multiple variables that are spread-in here.
```

Comment:
Yes, good suggestion! I agree with splitting it up into multiple slices. That will probably make each file more readable and comprehensive. Do you want to do this as part of this PR?

Reply:
Yes, good suggestion! I agree with splitting it up into multiple slices. That will probably make each file more readable and comprehensive. 

Since we have already a lot of changes in this PR, you can address this in an upcoming PR. I created issue https://github.com/datavisyn/reprovisyn/issues/311 for it.